juriscraper/AbstractSite.py

```diff
@@ -77,7 +77,7 @@ def __init__(self, cnt=None):
 
     def __str__(self):
         out = []
-        for attr, val in self.__dict__.items():
+        for attr, val in list(self.__dict__.items()):
```

Comment:
`2to3` will change `for x, y in dict.items()` to `for x, y in list(dict.items()` but the former is still correct and preferable in Python 3 - the tool is just being overly conversative. It's because we're immediately consuming the iterator, which `2to3` doesn't realise.

Reply:
That's mea culpa. I was going to create a follow-up issue to go clean up all the `2to3`-isms, and establish a guide for contributors to feel free to use some py3isms on a go forward basis. I'm thinking primarily f-strings since there is much string construction and munging in this project, etc. 

Happy to help root out some of those `list()` wrapped iterators and generators if we want to update that before this goes live for 2.0.0

Reply:
That would be great. @cgdeboer

Reply:
https://github.com/freelawproject/juriscraper/pull/344 resolves the majority of the most obvious unecessary `list()` wrappers.

/cc @drewsilcock @flooie 

*(and looks like @mlissner just brought it in)*